ecs-init/Godeps/Godeps.json

```diff
@@ -15,8 +126,78 @@
 			"Rev": "c9ad0ce23f68428421adfc6ced9e6123f54788a5"
 		},
 		{
+			"ImportPath": "github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus",
```

Comment:
go-dockerclient does not have a `vendor` directory.  This old version of go-dockerclient has an `external` directory that is not excluded because it's called `external` (and the imports are all rewritten within go-dockerclient to use that `external` package), so I don't think we're running into https://github.com/tools/godep/issues/498.

Is there not a newer version of go-dockerclient that (1) does not have a `vendor` (or an `external`) package and (2) works with Go 1.5?